docs/source/quickstart.rst

```diff
@@ -1,2 +1,154 @@
-Quickstart
-==========
+.. _quickstart:
+
+#############
+ Quick Start
+#############
+
+This page will guide you to train a GAN to remove glasses for human
+faces, and a DDPM to add glasses on human faces.
+
+.. _quickstart-install:
+
+**************
+ Installation
+**************
+
+Clone this repo:
+
+.. code:: bash
+
+   git clone --recursive https://github.com/jolibrain/joliGEN.git
+   cd joliGEN
+
+Install PyTorch and other dependencies (torchvision, visdom) with:
+
+.. code:: bash
+
+   pip install -r requirements.txt --upgrade
+
+*****************************************************
+ Train a GAN for unpaired dataset and object removal
+*****************************************************
+
+.. _quickstart-gan-dataset:
+
+Download the Dataset
+====================
+
+Download the dataset (3.3 Gb) and unzip it:
+
+.. code:: bash
+
+   wget https://www.deepdetect.com/joligen/datasets/noglasses2glasses_ffhq.zip
+   unzip noglasses2glasses_ffhq.zip
+
+Train your GAN
+==============
+
+We will train a GAN based on `CUT
+<https://github.com/taesungp/contrastive-unpaired-translation>`_ to
+remove glasses from human's faces.
+
+Use ``train.py`` along with the `example CUT config file
+<https://github.com/jolibrain/joliGEN/examples/example_glasses2noglasses.json>`_
```

Comment:
URL is wrong!

Reply:
It will be functionnal when this branch will be merged into master